charts/load-test-http/values.yaml

```diff
@@ -1,54 +1,20 @@
-### Defaults values for the load-test-http experiment chart.
-
-### The documentation follows Helm recommendations described in the URL below.
-### https://helm.sh/docs/chart_best_practices/values/#document-valuesyaml
+### Parameters of the the load-test-http experiment chart and the default values.
 
 ##################################
 
-### url   HTTP(S) URL where the app receives GET or POST requests.
-### This parameter is required.
-# url: null
-
-### headers   HTTP headers to be used in requests sent to the app.
-### Specified as map[string]string
-# headers: null
-
-### numQueries    Number of requests sent to the app. 
-# numQueries: null
-
-### duration    Duration for which requests are sent to the app.
-### Value can be any Go duration string (https://pkg.go.dev/maze.io/x/duration#ParseDuration)
-### This field is ignored if `numQueries` is specified.
-# duration: null
+### This chart accepts all parameters that are part of the collectHTTPInputs struct.
+### https://github.com/iter8-tools/iter8/blob/2891c13aac668bd651cc8e76f2d11a9d9a4fb75b/base/collect_http.go#L25
```

Comment:
This is the permalink Iter8 gives, so unless, the repo is removed, I believe the link will always be valid and refer to the same thing, which is the effect we  want?
***
Once we created a tagged release, I can follow up with another PR to update the perma link.

Reply:
I guess if we want to keep updating this link with each release we can do that. The link remains valid since it points to a particular commit. However, the code might change after this commit and we want the documentation to point at the recent version. I know. It is version specific. But I am thinking about maintenance here. I'm willing to trade off what someone looking at a particular version might see.

Reply:
The only reason to change the permalink would be that the underlying struct (and hence the supported parameters/default values) have changed.
***
Until then, they can look at a fixed link given in the values.yaml file, even if it is not the latest.

Reply:
The idea is *not* to change on every release. But to change on every change to the struct/values.yaml.